.ci/renewable-deploy/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -ex
+
+.ci/renewable-deploy/variable-validator.sh
+
+# edit configuration from ENV variables
+docker run \
+  -e DOCKER_IMAGE_TAG_PHP_FPM \
+  -e DOCKER_IMAGE_TAG_ELASTIC \
+  -e DOCKER_USERNAME \
+  -e FIRST_DOMAIN_HOSTNAME \
+  -e SECOND_DOMAIN_HOSTNAME \
+  -e THIRD_DOMAIN_HOSTNAME \
+  -e GOOGLE_CLOUD_PROJECT_ID \
+  -e GKE_LOAD_BALANCER_IP \
+  -v ${WORKSPACE}:/code \
+  -w /code \
+  --rm \
+  s3tezsky/yq:0.1 \
```

Comment:
Do you pledge to maintain this image for eternity and forever and ever?

Reply:
I belive that shopsys will live much longer than me. So maybe we could move it into shopsys organization?

Reply:
I should try to use https://github.com/shopsys/kubernetes-buildpack

Reply:
changed in https://github.com/shopsys/demoshop/pull/84/commits/7e4f93556f099f3b1b02df718e4b2c8ced6cbeb1